x/staking/module.go

```diff
@@ -172,6 +178,37 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 	return EndBlocker(ctx, am.keeper)
 }
 
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(
+			provideModuleBasic,
+			provideModule,
+		),
+	)
+}
+
+func provideModuleBasic() runtime.AppModuleBasicWrapper {
+	return runtime.WrapAppModuleBasic(AppModuleBasic{})
+}
+
+type stakingInputs struct {
+	depinject.In
+
+	Config        *modulev1.Module
+	AccountKeeper types.AccountKeeper `key:"cosmos.auth.v1.AccountKeeper"`
+	BankKeeper    types.BankKeeper    `key:"cosmos.auth.v1.BankKeeper"`
```

Comment:
I think you might need to update x/bank/module.go so that `provideModule` returns a struct where you can define the key for the bank keeper (since how it's done in x/auth/module.go)

Reply:
As requested by @JeancarloBarrios, I pushed a small commit c0eaec6981734f34d1f5ff0fb1d453d7ae81029b to hopefully unblock other PRs that might need staking keeper in depinject

Reply:
We also need to return a struct in x/staking provideModule in order to define a key on the returned staking keeper

Reply:
Fixed!